.storybook/locale.ts

```diff
@@ -339,10 +339,7 @@ let locale: Record<string, string> = {
   braveTodayOptInActionLabel: 'Show Brave News',
   braveTodayOptOutActionLabel: 'No thanks',
   braveTodayScrollHint: 'Scroll for Brave News',
-  editCardsTitle: 'Edit Cards',
-  braveWalletBuyWithWyre: 'Buy with Wyre',
-  braveWalletBuyWithRamp: 'Buy with Ramp',
-  braveWalletBuyContinueButton: 'Continue'
```

Comment:
Noticed this change, was this intentional?
If so, it looks like `braveWalletBuyContinueButton` has a different string value in `brave_wallet/stories/locale.ts`